Resources/views/macros.html.twig

```diff
@@ -0,0 +1,45 @@
+{% macro toast(message,type,role = 'status',live = 'polite',delay = 10000, autohide) %}
+
+  {% set icon = 'check' %}
+  {% if type == 'success' %}
+    {% if autohide is null or autohide is not defined %}
+      {% set autohide = true %}
+    {% endif %}
+  {% endif %}
+  {% if type == 'info' %}
+    {% set icon = 'info-circle' %}
+    {% if autohide is null or autohide is not defined %}
+      {% set autohide = true %}
+    {% endif %}
+  {% endif %}
+  {% if type == 'warning' %}
+    {% set icon = 'exclamation' %}
+    {% if autohide is null or autohide is not defined %}
+      {% set autohide = false %}
+    {% endif %}
+  {% endif %}
+  {% if type == 'danger' %}
```

Comment:
I should set all alerts default on aria live assertive and role alert. That are also the guidelines of bootstrap, are do you have other thoughts about it?

Reply:
Also I don't think it need to be something that can be changed by the developer, maybe just my meaning. In my eyes it's something that is determined by the type of error.